### PR TITLE
Add configuration options to README

### PR DESCRIPTION
**Pull request by martialblog**
A small addition to the README, so that the available config options are included.

See #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,26 @@ The default values are used internally, but might to be adjusted for your specif
...
+
+Available configuration options:
+
+```yaml
+linters-settings:
+  funlen:
+    # Checks the number of lines in a function.
+    # If lower than 0, disable the check.
+    # Default: 60
+    lines: -1
```
Comment (robinknaapen) [inline README.md]: Maybe set the values to the default settings instead of the disabled ones

**Review by robinknaapen — CHANGES_REQUESTED** [review]
After this LGTM

File: README.md (hunk 1 above)
Comment (martialblog) [reply README.md]: Makes sense. I just copied the entry from the golangci-lint examples.

**Review by robinknaapen — APPROVED** [review]

**Comment by robinknaapen** [comment]
@martialblog thanks for the contribution

**Pull request merged**